src/extensions/default/CodeFolding/main.js

```diff
@@ -83,13 +76,12 @@ define(function (require, exports, module) {
         if (!editor || !saveFolds) { return; }
 
         var cm = editor._codeMirror;
-        if (!cm) {return; }
```

Comment:
`editor._codeMirror` can never be null for any Editor, so I removed this check (the `editor` arg itself could be null -- e.g. in rare case where the `"documentRefreshed"` listener fires on a Document with no `_masterEditor` -- so I left that check above in place.